t/ui/18-tests-details.t

```diff
@@ -45,6 +45,35 @@ sub schema_hook {
     # set assigned_worker_id to test whether worker still displayed when job set to done
     # manually for PhantomJS test
     $jobs->find(99963)->update({assigned_worker_id => 1});
+
+    my $new_job = $jobs->create(
+        {
+            id                 => 100001,
+            group_id           => 1001,
+            result             => "none",
+            state              => "waiting",
```

Comment:
So now the true question: how would a job get into this state?

Reply:
Yes, that hasn't been implemented yet. Somehow isotovideo should update the state I guess.

Reply:
I don't think so. We want the job to be locked for a developer - if isotovideo is waiting or not is to be queried through WS.

Reply:
@coolo In the past we had `$status->{status}->{needinput}` which was [removed already](https://github.com/os-autoinst/openQA/pull/1632/files#diff-85ae48e70a5c110c9e439c3a5ea28d5fL1467). My initial thought was:

* Check if current_module is equal to PAUSE_AT (I forgot the name of the variable again) if so, then send the query to the worker to check if it already started to accept commands, and then update the status from the worker. And use a similar strategy as needinput used to do. 
* Also allow to stop waiting if there are no developer sessions running.

Reply:
Adding a job for waiting doesn't make any sense anymore.

I would actually add some UI-only test where we fake just some state by injecting JavaScript and test whether the page behaves correctly (right elements shown/hidden, status strings correctly formatted, ...). This is how I'd like to test https://github.com/os-autoinst/openQA/pull/1691/files (beside extending the developer full stack test).